init.py

```diff
@@ -150,7 +150,10 @@ def main() -> None:
         # $ csmith --version csmith 2.3.0
         # Git version: 30dccd7
         version = res.split("\n")[0].split()[1]
-        csmith["include_path"] = "/usr/include/csmith-" + version
+        if Path("/usr/include/csmith").exists():
+            csmith["include_path"] = "/usr/include/csmith"
+        else:
+            csmith["include_path"] = "/usr/include/csmith-" + version
```

Comment:
Maybe we can move this code to diopter's csmith generator?

Reply:
:+1:
A bit more validation on `diopter`s side might be nice. It already bit one user.